WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -49,6 +66,7 @@
 - [ ] Visit `http://theverge.com`, open any article, verify the reader mode icon is shown in the URL bar
 - [ ] Verify tapping on the reader mode icon opens the article in reader mode
 - [ ] Edit reader mode settings and open different pages in reader mode and verify if the setting is retained across each article
+- [ ] Verify reader mode works with tool bar on top enabled
```

Comment:
Suggestion

`Verify reader mode works when toolbar is set as Top Bar and Bottom Bar`